aws/import_aws_config_authorization_test.go

```diff
@@ -0,0 +1,30 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccConfigAuthorization_import(t *testing.T) {
```

Comment:
Minor nitpick: we're going to move away from separate import test files and tests (#4705). We can simply add the final TestStep the existing `_basic` test 👍